### DISCUSSION TITLE
References for electricity disaggregation

### DISCUSSION THREAD

**Discussion by MoLi7** [USEPA/useeior#274] [Q&A] [answered]
Hi Team,

Do we have formal references behind [these](https://github.com/USEPA/useeior/tree/master/inst/extdata/disaggspecs) disagg specs so that we can properly cite the work?

Thank you in advance 🙏,
Mo

**Comment by jvendries** [comment]
Hi Mo,

There isn't a formal reference yet for the electricity disaggregation specs. They are currently in development and subject to change, so we do not recommend their use.

Best, 

Jorge

**Reply by bl-young** [reply]
Yes and the model which uses those specs has not yet been formally reviewed and so is actually going to be removed in the next release.

**Reply by sidpatelgit** [reply]
Thanks for the prompt replies here! When you say "has not yet been formally reviewed" - do you expect that after a formal review to come, you would add back the electricity disaggregation (or other disaggregations)? Or do you expect to permanently deprecate disaggregation?

**Reply by bl-young** [reply]
I'll ask @WesIngwersen to chime in on this.

**Reply by sidpatelgit** [reply]
A related question - what source data was used to come up with the disaggregation of the electricity sector? Was it eGRID?

**Reply by jvendries** [reply]
> A related question - what source data was used to come up with the disaggregation of the electricity sector? Was it eGRID?

eGrid and EIA data was used for creating the disaggregation specs currently on the repository, but it is worth noting again that these values are very much subject to change.

**Reply by sidpatelgit** [reply]
Very helpful thank you!

**Reply by WesIngwersen** [reply, marked as answer]
Thanks for your interest. We have not yet defined an EPA product (which gets reviewed) that will include the disaggregation of detailed electricity by generation energy sources + transmission/distribution and it has not been through formal review. But it is a frequently  requested enhancement so we also will keep it in the cue for potential implementations.

Just as a reminder, any code here, whether or not in a release, can be used in accordance with the very permissive [MIT license](https://github.com/USEPA/useeior/?tab=MIT-1-ov-file#readme), but please see again our [Disclaimer at the bottom of the readme](https://github.com/USEPA/useeior/?tab=readme-ov-file#disclaimer).

**Reply by MoLi7** [reply]
Thank you, team! @WesIngwersen @jvendries @bl-young
